### PR TITLE
Eslint rules order

### PR DESCRIPTION
**Pull request by sabau**
_Description:_

- Reorder eslint to maximize the effect of vue:recommended

- making green this PR takes few minutes, but it will change most of vue files. To minimize conflits we decided to apply this after web merges into develop

❤️ Thank you!

---

- [ ] Added entries in `CHANGELOG.md` with issue # and GitHub username
- [ ] Reviewed `Files changed` in the github PR explorer

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -97,6 +97,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - [\#1379](https://github.com/cosmos/voyager/issues/1379) Changed some tests so they don't display errors on the console to better identify real errors @faboweb
 - [\#1792](https://github.com/cosmos/voyager/pull/1792) removed mocked demo mode @fedekunze
 - [\#1720](https://github.com/cosmos/voyager/issues/1720) Time format from 12 to 24h @sabau
+- change eslint rules order to maximise vue:recommended ones @sabau
```
Comment (fedekunze) [inline CHANGELOG.md]:
```suggestion
- [\#1840](https://github.com/cosmos/voyager/pull/1840) change eslint rules order to maximise vue:recommended ones @sabau
```

**Comment by sabau** [comment]
I restored the precommit if that can help

**Comment by sabau** [comment]
I found the eslint plugin, it seems the autofix on save is possible also here, it needs a flag for that:
```
eslint.autoFixOnSave - enables auto fix on save. Please note auto fix on save is only available if VS Code's files.autoSave is either off, onFocusChange or onWindowChange. It will not work with afterDelay
```

**Comment by jbibla** [comment]
@sabau a lot of conflicts here. is this still blocked? let's close and reopen when possible?

**Pull request closed without merging**